lib/watcher/appmetrics.js

```diff
@@ -0,0 +1,76 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: strong-supervisor
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var UNUSED_PROBES = [
+  'eventloop',
+  'profiling',
+  'http',
+  'mongo',
+  'socketio',
+  'mqlight',
+  'postgresql',
+  'mqtt',
+  'mysql',
+  'redis',
+  'memcached',
+  'oracledb',
+  'strong-oracle',
+  'requests',
+  'trace',
+];
+
+exports.worker = function(handle) {
+  if (!global.APPMETRICS) {
```

Comment:
I noticed that the adapter creates `global.APPMETRICS`. I'm not necessarily opposed to finding another way, but would this be sufficient for disabling this watcher if appmetrics isn't present?